doc/release-notes/0.9/0.9.md

```diff
@@ -0,0 +1,228 @@
+<tr><td valign="top"><a href="https://github.com/eclipse/openj9/issues/1699">#1699</a></td>
+<td valign="top">Specify heap limits as a percentage of total memory</td>
+<td valign="top">All platforms</td>
+<td valign="top">Ability to easily set heap limits with the <b>-XX:MaxRAMPercentage</b> and <b>-XX:InitialRAMPercentage</b> options instead of statically setting them with the <b>-Xmx</b> and <b>-Xms</b> options.</td>
+</tr>
+<!-- Remove? Confirm.<tr><td valign="top"><a href="https://github.com/eclipse/openj9/issues/1776">#1776</a></td>
+<td valign="top">Extra page of memory no longer required when specifying an object heap size that is a multiple of the page size</td>
+<td valign="top">Linux on Z and z/OS</td>
+<td valign="top">This change avoids wasting memory or calculating a heap size that is 16 bytes less than the page size.</td>
+</tr>-->
+<tr><td valign="top"><a href="https://github.com/eclipse/openj9/issues/1668">#1668</a></td>
+<td valign="top">Idle tuning feature enabled</td>
+<td valign="top">Linux on Z, Linux on Power systems</td>
+<td valign="top">Ability to maintain a minimal memory footprint during the life time of an application.</td>
+</tr>
+<tr><td valign="top"><a href="https://github.com/eclipse/openj9/issues/1517">#1517</a></td>
+<td valign="top">Changes to the default Java heap size for applications that run inside a container</td>
+<td valign="top">Linux</td>
+<td valign="top">By default, 75% of the physical memory is allocated for the maximum Java heap setting when the <b>-XX:+UseContainerSupport</b> option is set.</td>
+</tr>
+<tr><td valign="top"><a href="https://github.com/eclipse/openj9/issues/1500">#1500</a></td>
+<td valign="top">Support for nested jar files</td>
+<td valign="top">All platforms</td>
+<td valign="top">Improvements to the <code>com.ibm.oti.shared</code> API for manipulating nested jar files, such as used by Spring.</td>
+</tr>
+<tr><td valign="top"><a href="https://github.com/eclipse/openj9/issues/1786">#1786</a></td>
+<td valign="top">Changes to OpenJ9 <code>java.lang.String</code> class and use of OpenJDK <code>StringBuffer</code> and <code>StringBuilder</code></td>
+<td valign="top">OpenJDK 9+, all platforms</td>
+<td valign="top">To match the behavior of OpenJDK, <code>java.lang.String</code> no longer has a count field, which changes the way that <code>String.subString()</code> works compared to Java 8. Similarly <code>StringBuffer</code and <code>StringBuilder</code> do not share the value array with any <code>String</code> created by <code>toString()</code>.</td>
+</tr>
+
+<tr><td valign="top">
+<a href="https://github.com/eclipse/openj9/issues/1603">#1603</a>,
+<a href="https://github.com/eclipse/openj9/issues/1642">#1642</a>,
+<a href="https://github.com//eclipse/omr/issues/2494">eclipse/omr#2494</a>,
+<a href="https://github.com//eclipse/omr/issues/2524">eclipse/omr#2524</a>
+</td>
+<td valign="top">Full support for shutdown signals</td>
+<td valign="top">All Linux variants, AIX, zOS</td>
```

Comment:
Seems weird to mention zOS when OpenJ9+extensions doesn't support it, but its not harmful.